dogfood/coder/main.tf

```diff
@@ -380,6 +396,24 @@ module "devcontainers-cli" {
   agent_id = coder_agent.dev.id
 }
 
+module "claude-code" {
+  count               = local.has_ai_prompt ? data.coder_workspace.me.start_count : 0
+  source              = "dev.registry.coder.com/coder/claude-code/coder"
```

Comment:
Does this have to be sourced from `dev.registry.coder.com`? Is this for dogfooding?

Reply:
We could use either, the module versions are published simultaneously. However, there doesn't seem to be much consistency between using `registry.coder.com` and `dev.registry.coder.com` (see above module definitions).

Reply:
Ideally, we should always use dev.registry.coder.com so that we can catch any breaking changes before our customers do.